tests/gcp/hooks/test_bigquery.py

```diff
@@ -693,42 +762,75 @@ def run_with_config(config):
 
 
 class TestDatasetsOperations(unittest.TestCase):
+    @mock.patch(
+        'airflow.gcp.hooks.base.CloudBaseHook._get_credentials_and_project_id',
+        return_value=("CREDENTIALS", "PROJECT_ID",)
+    )
+    @mock.patch("airflow.gcp.hooks.bigquery.BigQueryHook.get_service")
+    def test_create_empty_dataset_no_dataset_id_err(self, mock_get_service, mock_get_creds_and_proj_id):
+        with self.assertRaisesRegex(ValueError, r"{} not provided datasetId\. Impossible to create dataset"):
```

Comment:
```
        with self.assertRaisesRegex(ValueError, r"?? not provided datasetId\. Impossible to create dataset"):
```
Is there good regexp in the marked place? It doesn't look good to me.

Reply:
I agree it looks weird but the **Exception** message looks exactly like that. I've improved this message and adjusted regex as well.